### PR TITLE
Move report models and fragments to core

### PR DESCRIPTION
**Pull request by balajimanoharan**
- Move model fragments from navi-reports to core
- Move models from navi-reports to core

TODO
- Move dashboard models to core

### CODE REVIEW

File: packages/core/tests/unit/models/bard-request/fragments/filter-test.js [hunk 1]
```diff
@@ -1,20 +1,23 @@
-import Ember from 'ember';
 import { moduleForModel, test } from 'ember-qunit';
-import { setupMock, teardownMock } from '../../../../helpers/mirage-helper';
+import { startMirage } from '../../../../../initializers/ember-cli-mirage';
 import wait from 'ember-test-helpers/wait';
 import config from 'ember-get-config';
+import { A } from '@ember/array';
```
Comment (mattgraham1995) [inline packages/core/tests/unit/models/bard-request/fragments/filter-test.js]: `A as arr`

File: packages/dashboards/tests/unit/models/dashboard-test.js [hunk 2]
```diff
@@ -134,22 +134,25 @@ test('Cloning Dashboards', function(assert) {
...
-          assert.deepEqual(rule,
-            Store.peekRecord('deliveryRule', 1),
-            'deliveryRule is fetched for current user');
-        });
-      });
-    });
-  });
-});
+/**
+ * TODO Fix test after moving to core
```
Comment (cythrawll) [inline packages/dashboards/tests/unit/models/dashboard-test.js]: Why is this test commented out? can we fix it now?

Comment (mattgraham1995) [reply packages/dashboards/tests/unit/models/dashboard-test.js]: It's part of my task of moving the dashboards models to core. This PR is just for the reports models

File: packages/core/tests/unit/models/bard-request/fragments/filter-test.js (hunk 1 above)
Comment (balajimanoharan) [reply packages/core/tests/unit/models/bard-request/fragments/filter-test.js]: Done!

**Pull request merged**